src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+type Props = {
+  sex: string;
+  slug: string;
+  name: string;
+};
+
+export const PersonLink: React.FC<Props> = ({
+  sex,
+  slug,
+  name,
+}) => {
+  return (
+    <Link
+      className={classNames({
+        'has-text-danger': sex === 'f',
```

Comment:
The condition 'sex === 'f'' assumes that 'f' represents female. Ensure that this matches the data structure and requirements. If 'f' is incorrect, adjust this condition accordingly.